addons/web/static/src/views/form/status_bar_buttons/status_bar_buttons.xml

```diff
@@ -9,7 +9,7 @@
                 <t t-if="firstSlot" t-slot="{{ firstSlot }}" />
                 <Dropdown t-if="otherSlots.length">
                     <button class="btn btn-secondary" title="More">
-                        <i class="fa fa-fw fa-ellipsis-v" />
```

Comment:
Why not replacing with `oi-fw` ? 
In fact, the button is a bit smaller using  `oi-fw`, which would be better since this button is on mobile.

| `oi` | not `oi` |
|--------|--------|
| ![Capture d’écran 2025-04-07 à 11 22 35](https://github.com/user-attachments/assets/aab331bb-aa6e-42a5-8cfc-b8e8702db03e) | ![Capture d’écran 2025-04-07 à 11 23 22](https://github.com/user-attachments/assets/c6013eff-fc49-44eb-b48f-a2875a638db2) |